src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,40 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  selectedTodo: Todo | null,
```

Comment:
```suggestion
  selectedTodo: Todo,
```
You render the modal only if you actually have selectedTodo